notebooks/api/0.8/06-domain-register-control-flow.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
NITPICK: Could we rename the notebook to `07-domain-register-control-flow` as we already have `06`

Reply:
done